packages/mpv.rb

```diff
@@ -3,64 +3,93 @@
-  source_sha256 '9163f64832226d22e24bbc4874ebd6ac02372cd717bef15c28a0aa858c5fe592'
-
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.32.0-chromeos-x86_64.tar.xz',
+  source_url "https://github.com/mpv-player/mpv/archive/v#{version}.tar.gz"
+  source_sha256 'f1b9baf5dc2eeaf376597c28a6281facf6ed98ff3d567e3955c95bf2459520b4'
 
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mpv-0.33.0-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: 'ae9f812ccb3239ca85d25a6e919e1ded6e459867ba00d2027ea832782a38f9f1',
-     armv7l: 'ae9f812ccb3239ca85d25a6e919e1ded6e459867ba00d2027ea832782a38f9f1',
-       i686: '41f96ce28132e2d60978c3e7450594b58b03fda67edbb8fa2e56ce6442d3d1ee',
-     x86_64: 'e9623b3fdd06ab5bb762a15a3d7ef9b7875a10d4f0a5baaed5f83b9250b9fbe4',
-
+  binary_sha256({
+    aarch64: '834b7660f6c44bc5a52236828f6451e03da7f80ce7dcaef84aa5bf539209a0fb',
+     armv7l: '834b7660f6c44bc5a52236828f6451e03da7f80ce7dcaef84aa5bf539209a0fb',
+       i686: 'e0bdd8907d0f3673397adcc6d042e878510a6657e347fcc4c30db9bdf40af33d',
+     x86_64: 'ae78493b16e21e2c9b8ac6ff794a4a89305f9f3686392961b4277bcd5175d109'
   })
 
-  depends_on 'ld_default' => :build
+  depends_on 'alsa_lib'
   depends_on 'docutils'
   depends_on 'ffmpeg'
-  depends_on 'gdbm'
+  depends_on 'jack'
   depends_on 'lcms'
   depends_on 'libarchive'
+  depends_on 'libass'
+  depends_on 'libbluray'
   depends_on 'libcaca'
   depends_on 'libcdio'
+  depends_on 'libcdio_paranoia'
+  depends_on 'libdrm'
+  depends_on 'libdvdnav'
+  depends_on 'libdvdread'
+  depends_on 'libjpeg'
+  depends_on 'libjpeg_turbo'
+  depends_on 'libva'
+  depends_on 'libvdpau'
+  depends_on 'libx11'
+  depends_on 'libxext'
+  depends_on 'libxinerama'
+  depends_on 'libxkbcommon'
+  depends_on 'libxrandr'
+  depends_on 'libxss'
   depends_on 'libxv'
   depends_on 'luajit'
+  depends_on 'mesa'
+  depends_on 'mujs'
+  depends_on 'pipewire'
   depends_on 'pulseaudio'
-  depends_on 'uchardet'
-  depends_on 'xdg_base'
+  depends_on 'rubberband'
+  depends_on 'shaderc'
   depends_on 'sommelier'
+  depends_on 'vulkan_headers' => :build
+  depends_on 'vulkan_icd_loader'
+  depends_on 'wayland'
+  depends_on 'xdg_base'
+  depends_on 'zimg'
 
   def self.build
-    # Use the gold linker.
-    old_ld = `ld_default g`.chomp
-    system 'pip3 install docutils'
+    system 'pip install docutils'
```

Comment:
Use `depends_on 'docutils' => :build` instead?

Reply:
Sure. I wish there was a way to get :build depends to get uninstalled after a build...

Reply:
Sure. I wish there was a way to get :build depends to get uninstalled after a build...

Reply:
I see this conversation was marked solved.  Does that mean `depends_on 'docutils' => :build` doesn't work as expected?

Reply:
`depends_on 'docutils' => :build` is in there now.

Reply:
(and it works)

Reply:
Does it work without `system 'pip install docutils'`?  Shouldn't that be removed?

Reply:
Ah! I missed that! Let me take those out...